packages/ui/quantity/Quantity.tsx

```diff
@@ -0,0 +1,22 @@
+import { useCounter } from '@mantine/hooks';
+import { BsDash, BsPlus } from 'react-icons/bs';
+
+import { Count, QuantityWrapper, SquareButton } from './style';
+
+export function Quantity() {
+  const [count, handlers] = useCounter(0, { min: 0, max: 100 });
+
+  return (
+    <QuantityWrapper>
+      <SquareButton onClick={handlers.decrement}>
+        <BsDash style={{ margin: '0', padding: '0' }} />
+      </SquareButton>
```

Comment:
Inline Styling - Try this instead:
```suggestion
      import { BsDashStyled } from './style';
      <SquareButton onClick={handlers.decrement}>
        <BsDashStyled />
      </SquareButton>
```